pyart/correct/noise.py

```diff
@@ -0,0 +1,99 @@
+"""
+pyart.correct.noise
+===================
+
+Corrects polarimetric variables for noise
+
+.. autosummary::
+    :toctree: generated/
+
+    correct_noise_rhohv
+
+"""
+
+import numpy as np
+
+from ..config import get_metadata, get_field_name, get_fillvalue
+
+
+def correct_noise_rhohv(radar, urhohv_field=None, snr_field=None,
+                        zdr_field=None, nh_field=None, nv_field=None,
+                        rhohv_field=None):
+    """
+    Corrects RhoHV for noise
+
+    Parameters
+    ----------
+    radar : Radar
+        radar object
+
+    urhohv_field : str
+        name of the RhoHV uncorrected for noise field
+
+    snr_field, zdr_field, nh_field, nv_field: str
+        names of the SNR, ZDR, horizontal channel noise in dBZ and vertical
+        channel noise in dBZ used to correct RhoHV
+
+    rhohv_field: str
+        name of the rhohv field to output
+
+    Returns
+    -------
+    rhohv : dictionary of dictionaries
+        noise corrected RhoHV field
+
+    """
+    # parse the field parameters
+    if urhohv_field is None:
+        rhohv_field = get_field_name('uncorrected_cross_correlation_ratio')
```

Comment:
Should this be urhohv_field = get_field_name('uncorrected_cross_correlation_ratio')